myapp/app/controllers/application_controller.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
 class ApplicationController < ActionController::Base
+  before_action :render_503, if: :maintenance_mode?
```

Comment:
[修正任意]
・・503 ではなく、
```suggestion
  before_action :render_service_temporarily_unavailable, if: ・・・
```
と長くなっても具体的な名称のが良いかもです。